rotkehlchen/globaldb/handler.py

```diff
@@ -135,6 +136,19 @@ def __new__(
         GlobalDBHandler.__instance.conn = _initialize_global_db_directory(data_dir, sql_vm_instructions_cb)  # noqa: E501
         return GlobalDBHandler.__instance
 
+    @property
+    def packaged_db_conn(self) -> DBConnection:
```

Comment:
We prefer against using properties because they tend to hide code complexity. If you see it might look like a normal attribute but it could have slow logic behind